src/test/java/pages/CheckoutLoginPage.java

```diff
@@ -3,6 +3,7 @@
 import com.codeborne.selenide.SelenideElement;
 import static com.codeborne.selenide.Selenide.$x;
 
+
 public class CheckoutLoginPage extends BasePage{
     private static SelenideElement checkoutAsGuestBtn = $x("//input[@class='button-1 checkout-as-guest-button']");
```

Comment:
ненадёжный локатор, лучше заменить его на 
```suggestion
    private static SelenideElement checkoutAsGuestBtn = $x("//input[contains(concat(' ', @class, ' '), ' checkout-as-guest-button ')];
```
пробел перед и после класса очень важен, без этого локатор будет находить элементы с классами dssdf-checkout-as-guest-button или checkout-as-guest-button-dsfsdfsdfsd